src/plugins/menu/src/components/loading_screen.rs

```diff
@@ -0,0 +1,62 @@
+use crate::traits::{get_node::GetNode, instantiate_content_on::InstantiateContentOn, LoadUi};
+use bevy::prelude::*;
+use loading::traits::progress::{AssetLoadProgress, DependencyResolveProgress, Progress};
+use std::marker::PhantomData;
+
+#[derive(Component)]
+pub(crate) struct LoadingScreen<T>(PhantomData<T>)
+where
+	T: Progress;
+
+impl<T> LoadUi<AssetServer> for LoadingScreen<T>
+where
+	T: Progress,
+{
+	fn load_ui(_: &mut AssetServer) -> Self {
+		LoadingScreen(PhantomData)
+	}
+}
+
+impl<T> GetNode for LoadingScreen<T>
+where
+	T: Progress,
+{
+	fn node(&self) -> NodeBundle {
+		NodeBundle {
+			style: Style {
+				width: Val::Vw(100.),
+				height: Val::Vh(100.),
+				flex_direction: FlexDirection::ColumnReverse,
+				padding: UiRect::bottom(Val::Px(100.)).with_left(Val::Px(50.)),
+				..default()
+			},
+			background_color: Color::BLACK.into(),
+			z_index: ZIndex::Global(i32::MAX),
+			..default()
+		}
+	}
+}
+
+impl InstantiateContentOn for LoadingScreen<AssetLoadProgress> {
+	fn instantiate_content_on(&self, parent: &mut ChildBuilder) {
+		parent.spawn(TextBundle::from_section(
+			"Loading Assets ...",
+			TextStyle {
+				font_size: 32.,
+				..default()
+			},
+		));
```

Comment:
`TextStyle` from bevy is made with ease of use in mind, as far as I know. So it defaults to a render-able font. Introducing a new font at this point doesn't make sense, since we currently always use the default font. This might change in the future and is thus potentially subject to future PRs.